Problem with building with maven 3.8.2

Since maven 3.8.2 (which includes https://github.com/apache/maven/commit/5a8997312680a4b9c69a129801524691bc546c08 that I think is the cause of change) the EL can no longer be built due to [directory-maven-plugin](https://github.com/jdcasey/directory-maven-plugin) not being compatible (and BTW probably dead anyway).

It results with:
```
[ERROR] Failed to execute goal org.commonjava.maven.plugins:directory-maven-plugin:0.3.1:highest-basedir (find-project-root) on project org.eclipse.persistence.parent: Cannot find a single highest directory for this project set. First two candidates directories don't share a common root. -> [Help 1]
```

As maven versions are not specified exactly in 
https://github.com/eclipse-ee4j/eclipselink/blob/c0f808e3204abeafeaf9c63e69b16d0e0f75571f/.github/workflows/codeql-analysis.yml#L57  or in https://github.com/eclipse-ee4j/eclipselink/blob/c0f808e3204abeafeaf9c63e69b16d0e0f75571f/etc/jenkins/pr_verify.groovy#L94 they might be upgraded out of control of this project.

Please consider taking action before that happens.

> Do you consider this to be a regression or just surfaced a bug in the mentioned plugin?

As [MNG-7170](https://issues.apache.org/jira/browse/MNG-7170) is classified as improvement, I'd think that this is regression in maven, unexpected in microrelease (3.8.1 to 3.8.2). 

[maven-3.8.2](https://issues.apache.org/jira/projects/MNG/versions/12349965) declares
> Fixes and non-invasive backports from Maven 4.0.0-alpha-1.

I have not investigated the plugin itself, so have no idea what happens [there](https://github.com/jdcasey/directory-maven-plugin/blob/directory-maven-plugin-0.3.1/src/main/java/org/commonjava/maven/plugins/execroot/HighestBasedirGoal.java#L110-L124) to result with:
```
Caused by: org.apache.maven.plugin.MojoExecutionException: Cannot find a single highest directory for this project set. First two candidates directories don't share a common root.
    at org.commonjava.maven.plugins.execroot.HighestBasedirGoal.findDirectory (HighestBasedirGoal.java:121)
    at org.commonjava.maven.plugins.execroot.AbstractDirectoryGoal.execute (AbstractDirectoryGoal.java:80)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
```
But this works with 3.8.1 and it doesn't break the build.

Nothing related seems to be mentioned in [known-issues](https://maven.apache.org/docs/3.8.2/release-notes.html#known-issues) yet.

--- a/bundles/eclipselink/pom.xml
+++ b/bundles/eclipselink/pom.xml
@@ -190,6 +190,14 @@
         <!-- included artifacts -->
         <dependency>
             <groupId>org.eclipse.persistence</groupId>
+            <artifactId>org.eclipse.persistence.parent</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+            <classifier>common-license</classifier>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.persistence</groupId>
             <artifactId>org.eclipse.persistence.asm</artifactId>
             <scope>provided</scope>
         </dependency>

--- a/bundles/eclipselink/src/main/assembly/eclipselink.zip.xml
+++ b/bundles/eclipselink/src/main/assembly/eclipselink.zip.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -22,6 +22,21 @@
     </formats>
     <includeBaseDirectory>false</includeBaseDirectory>
     <dependencySets>
+        <!--License files-->
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <scope>provided</scope>
+            <outputDirectory>eclipselink/</outputDirectory>
+            <includes>
+                <include>${project.groupId}:org.eclipse.persistence.parent:zip:common-license</include>
+            </includes>
+            <unpack>true</unpack>
+            <unpackOptions>
+                <includes>
+                    <include>*.html</include>
+                </includes>
+            </unpackOptions>
+        </dependencySet>
         <dependencySet>
             <useProjectAttachments>true</useProjectAttachments>
             <outputDirectory>eclipselink/jlib/</outputDirectory>
@@ -112,16 +127,6 @@
         </dependencySet>
     </dependencySets>
     <fileSets>
-        <!--License files-->
-        <fileSet>
-            <directory>${project.root.location}</directory>
-            <outputDirectory>eclipselink/</outputDirectory>
-            <includes>
-                <include>about.html</include>
-                <include>license.html</include>
-                <include>readme.html</include>
-            </includes>
-        </fileSet>
         <!--Scripts to bin directory-->
         <fileSet>
             <directory>${project.build.scriptSourceDirectory}</directory>

--- a/bundles/moxy-standalone/pom.xml
+++ b/bundles/moxy-standalone/pom.xml
@@ -37,6 +37,14 @@
     <dependencies>
         <dependency>
             <groupId>org.eclipse.persistence</groupId>
+            <artifactId>org.eclipse.persistence.parent</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+            <classifier>common-license</classifier>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.persistence</groupId>
             <artifactId>org.eclipse.persistence.moxy</artifactId>
         </dependency>
         <dependency>

--- a/bundles/moxy-standalone/src/main/assembly/moxy.xml
+++ b/bundles/moxy-standalone/src/main/assembly/moxy.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2020, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -22,6 +22,21 @@
     </formats>
     <includeBaseDirectory>false</includeBaseDirectory>
     <dependencySets>
+        <!--License files-->
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <scope>provided</scope>
+            <outputDirectory>moxy</outputDirectory>
+            <includes>
+                <include>${project.groupId}:org.eclipse.persistence.parent:zip:common-license</include>
+            </includes>
+            <unpack>true</unpack>
+            <unpackOptions>
+                <includes>
+                    <include>*.html</include>
+                </includes>
+            </unpackOptions>
+        </dependencySet>
         <dependencySet>
             <useProjectArtifact>false</useProjectArtifact>
             <outputDirectory>moxy/jlib/</outputDirectory>
@@ -29,16 +44,6 @@
         </dependencySet>
     </dependencySets>
     <fileSets>
-        <!--License files-->
-        <fileSet>
-            <directory>${project.root.location}</directory>
-            <outputDirectory>moxy/</outputDirectory>
-            <includes>
-                <include>about.html</include>
-                <include>license.html</include>
-                <include>readme.html</include>
-            </includes>
-        </fileSet>
         <!--Scripts to bin directory-->
         <fileSet>
             <directory>${generated.res.dir}/xsds</directory>

--- a/bundles/others/pom.xml
+++ b/bundles/others/pom.xml
@@ -53,6 +53,14 @@
         <!--Other modules-->
         <dependency>
             <groupId>org.eclipse.persistence</groupId>
+            <artifactId>org.eclipse.persistence.parent</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+            <classifier>common-license</classifier>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.persistence</groupId>
             <artifactId>eclipselink</artifactId>
         </dependency>
         <dependency>

--- a/bundles/others/src/main/assembly/eclipselink-plugins-nosql.zip.xml
+++ b/bundles/others/src/main/assembly/eclipselink-plugins-nosql.zip.xml
@@ -21,10 +21,22 @@
         <format>zip</format>
     </formats>
     <includeBaseDirectory>false</includeBaseDirectory>
-    <componentDescriptors>
-        <componentDescriptor>src/main/assembly/common-license.xml</componentDescriptor>
-    </componentDescriptors>
     <dependencySets>
+        <!--License files-->
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <scope>provided</scope>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>${project.groupId}:org.eclipse.persistence.parent:zip:common-license</include>
+            </includes>
+            <unpack>true</unpack>
+            <unpackOptions>
+                <includes>
+                    <include>*.html</include>
+                </includes>
+            </unpackOptions>
+        </dependencySet>
         <dependencySet>
             <includes>
                 <include>org.eclipse.persistence:org.eclipse.persistence.nosql</include>

--- a/bundles/others/src/main/assembly/eclipselink-plugins.zip.xml
+++ b/bundles/others/src/main/assembly/eclipselink-plugins.zip.xml
@@ -21,10 +21,22 @@
         <format>zip</format>
     </formats>
     <includeBaseDirectory>false</includeBaseDirectory>
-    <componentDescriptors>
-        <componentDescriptor>src/main/assembly/common-license.xml</componentDescriptor>
-    </componentDescriptors>
     <dependencySets>
+        <!--License files-->
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <scope>provided</scope>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>${project.groupId}:org.eclipse.persistence.parent:zip:common-license</include>
+            </includes>
+            <unpack>true</unpack>
+            <unpackOptions>
+                <includes>
+                    <include>*.html</include>
+                </includes>
+            </unpackOptions>
+        </dependencySet>
         <dependencySet>
             <outputFileNameMapping>${artifact.artifactId}_${nightlyVersion}.${artifact.extension}</outputFileNameMapping>
             <includes>

--- a/bundles/others/src/main/assembly/eclipselink-test-src.zip.xml
+++ b/bundles/others/src/main/assembly/eclipselink-test-src.zip.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -21,9 +21,23 @@
         <format>zip</format>
     </formats>
     <includeBaseDirectory>false</includeBaseDirectory>
-    <componentDescriptors>
-        <componentDescriptor>src/main/assembly/common-license.xml</componentDescriptor>
-    </componentDescriptors>
+    <dependencySets>
+        <!--License files-->
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <scope>provided</scope>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>${project.groupId}:org.eclipse.persistence.parent:zip:common-license</include>
+            </includes>
+            <unpack>true</unpack>
+            <unpackOptions>
+                <includes>
+                    <include>*.html</include>
+                </includes>
+            </unpackOptions>
+        </dependencySet>
+    </dependencySets>
     <fileSets>
         <!--Ant build files from resources-->
         <fileSet>

--- a/jpa/eclipselink.jaxrs.test/pom.xml
+++ b/jpa/eclipselink.jaxrs.test/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -27,7 +27,7 @@
         <groupId>org.eclipse.persistence</groupId>
         <artifactId>org.eclipse.persistence.test.server.parent</artifactId>
         <version>3.0.3-SNAPSHOT</version>
-        <relativePath>../../pom_test_server.xml</relativePath>
+        <relativePath>../../testing/server/pom.xml</relativePath>
     </parent>
 
     <properties>

--- a/jpa/eclipselink.jpa.nosql.test/pom.xml
+++ b/jpa/eclipselink.jpa.nosql.test/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -27,7 +27,7 @@
         <groupId>org.eclipse.persistence</groupId>
         <artifactId>org.eclipse.persistence.test.server.parent</artifactId>
         <version>3.0.3-SNAPSHOT</version>
-        <relativePath>../../pom_test_server.xml</relativePath>
+        <relativePath>../../testing/server/pom.xml</relativePath>
     </parent>
 
     <properties>

--- a/jpa/eclipselink.jpa.test/pom.xml
+++ b/jpa/eclipselink.jpa.test/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -27,7 +27,7 @@
         <groupId>org.eclipse.persistence</groupId>
         <artifactId>org.eclipse.persistence.test.server.parent</artifactId>
         <version>3.0.3-SNAPSHOT</version>
-        <relativePath>../../pom_test_server.xml</relativePath>
+        <relativePath>../../testing/server/pom.xml</relativePath>
     </parent>
 
     <properties>

--- a/jpa/eclipselink.jpa.wdf.test/src/it/java/org/eclipse/persistence/testing/tests/wdf/jpa1/entitymanager/TestFind.java
+++ b/jpa/eclipselink.jpa.wdf.test/src/it/java/org/eclipse/persistence/testing/tests/wdf/jpa1/entitymanager/TestFind.java
@@ -92,7 +92,7 @@ public class TestFind extends JPA1Base {
     public void testNegativ() {
         final EntityManager em = getEnvironment().getEntityManager();
         try {
-            Object result = em.find(Employee.class, Integer.valueOf(17 + 4));
+            Object result = em.find(Employee.class, Integer.valueOf(17 + 12345));
             verify(result == null, "found something");
         } finally {
             closeEntityManager(em);

--- a/jpa/eclipselink.jpars.test/pom.xml
+++ b/jpa/eclipselink.jpars.test/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -27,7 +27,7 @@
         <groupId>org.eclipse.persistence</groupId>
         <artifactId>org.eclipse.persistence.test.server.parent</artifactId>
         <version>3.0.3-SNAPSHOT</version>
-        <relativePath>../../pom_test_server.xml</relativePath>
+        <relativePath>../../testing/server/pom.xml</relativePath>
     </parent>
 
     <properties>

--- a/pom.xml
+++ b/pom.xml
@@ -111,9 +111,8 @@
         <bundle.version>${project.version}.${build.qualifier}</bundle.version>
         <project.build.testReports.subdirectory>test-reports</project.build.testReports.subdirectory>
         <project.build.testReports.summaryFile>test-summary</project.build.testReports.summaryFile>
+        <project.build.licenceResourceDirectory>${project.build.directory}/license</project.build.licenceResourceDirectory>
 
-        <!--Common properties-->
-        <common.licenceFiles>LICENSE.md, NOTICE.md</common.licenceFiles>
         <!--Integration test sources/resources directory. Relative directory to ${project.basedir}
         !!!Can't start with / character!!!-->
         <integration.test.directory>src/it/</integration.test.directory>
@@ -137,7 +136,7 @@
         <!--Default test DB is Derby-->
         <test.properties.file>${user.home}/${test.derby.properties.file}</test.properties.file>
         <test.properties.fileName>${test.derby.properties.file}</test.properties.fileName>
-        <test.properties.default.directory>${project.root.location}/etc</test.properties.default.directory>
+        <test.properties.default.directory>${project.build.directory}/test-default-properties</test.properties.default.directory>
         <test.database>derby</test.database>
         <!--false: to start In-Memory database, true: not start (MySQL, Oracle)-->
         <test.skip.in-memory.db>false</test.skip.in-memory.db>
@@ -272,6 +271,11 @@
     <dependencyManagement>
         <dependencies>
             <!-- our artifacts TODO: this belongs to BOM pom-->
+            <dependency>
+                <groupId>org.eclipse.persistence</groupId>
+                <artifactId>org.eclipse.persistence.parent</artifactId>
+                <version>${project.version}</version>
+            </dependency>
             <dependency>
                 <groupId>org.eclipse.persistence</groupId>
                 <artifactId>org.eclipse.persistence.asm</artifactId>
@@ -1116,11 +1120,6 @@
                     <artifactId>xml-maven-plugin</artifactId>
                     <version>1.0.2</version>
                 </plugin>
-                <plugin>
-                    <groupId>org.commonjava.maven.plugins</groupId>
-                    <artifactId>directory-maven-plugin</artifactId>
-                    <version>0.3.1</version>
-                </plugin>
                 <!--This plugin (g:org.kuali.maven.plugins) is used instead typical org.codehaus.mojo:properties-maven-plugin:1.0.0
                     due bug: "Circular property definition" if expression like this is used in the Maven pom
                     "<server.testrunner.prefix>${modelname}/${modelname}_ejb/</server.testrunner.prefix>"
@@ -1201,17 +1200,77 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>org.commonjava.maven.plugins</groupId>
-                <artifactId>directory-maven-plugin</artifactId>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
                 <executions>
                     <execution>
-                        <id>find-project-root</id>
-                        <phase>validate</phase>
+                        <id>common-license</id>
+                        <phase>generate-resources</phase>
                         <goals>
-                            <goal>highest-basedir</goal>
+                            <goal>single</goal>
+                        </goals>
+                        <inherited>false</inherited>
+                        <configuration>
+                            <descriptors>
+                                <descriptor>src/main/assembly/common-license.xml</descriptor>
+                            </descriptors>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>test-defaults</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <inherited>false</inherited>
+                        <configuration>
+                            <descriptors>
+                                <descriptor>src/main/assembly/test-defaults.xml</descriptor>
+                            </descriptors>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack-legal-resource</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>unpack</goal>
                         </goals>
                         <configuration>
-                            <property>project.root.location</property>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>${project.groupId}</groupId>
+                                    <artifactId>org.eclipse.persistence.parent</artifactId>
+                                    <version>${project.version}</version>
+                                    <classifier>common-license</classifier>
+                                    <type>zip</type>
+                                    <outputDirectory>${project.build.licenceResourceDirectory}</outputDirectory>
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>unpack-test-properties</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>${project.groupId}</groupId>
+                                    <artifactId>org.eclipse.persistence.parent</artifactId>
+                                    <version>${project.version}</version>
+                                    <classifier>test-defaults</classifier>
+                                    <type>zip</type>
+                                    <outputDirectory>${test.properties.default.directory}</outputDirectory>
+                                </artifactItem>
+                            </artifactItems>
                         </configuration>
                     </execution>
                 </executions>
@@ -1307,8 +1366,8 @@
                         <configuration>
                             <resources>
                                 <resource>
-                                    <directory>${project.root.location}</directory>
-                                    <includes>${common.licenceFiles}</includes>
+                                    <directory>${project.build.licenceResourceDirectory}/md</directory>
+                                    <includes>*</includes>
                                     <targetPath>META-INF</targetPath>
                                 </resource>
                             </resources>
@@ -1788,8 +1847,8 @@
             </activation>
             <modules>
                 <!--Helper parent pom for server side test modules-->
-                <module>pom_test_server.xml</module>
-                <module>pom_oracle_test_server.xml</module>
+                <module>testing/server</module>
+                <module>testing/server-oracle</module>
                 <module>foundation/org.eclipse.persistence.core.test.framework</module>
                 <module>foundation/eclipselink.core.test</module>
                 <module>jpa/org.eclipse.persistence.jpa.test.framework</module>

--- a/sdo/eclipselink.sdo.test.server/pom.xml
+++ b/sdo/eclipselink.sdo.test.server/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -27,7 +27,7 @@
         <groupId>org.eclipse.persistence</groupId>
         <artifactId>org.eclipse.persistence.test.server.parent</artifactId>
         <version>3.0.3-SNAPSHOT</version>
-        <relativePath>../../pom_test_server.xml</relativePath>
+        <relativePath>../../testing/server/pom.xml</relativePath>
     </parent>
 
     <properties>

--- a/src/main/assembly/common-license.xml
+++ b/src/main/assembly/common-license.xml
@@ -1,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0 which is available at
+    http://www.eclipse.org/legal/epl-2.0,
+    or the Eclipse Distribution License v. 1.0 which is available at
+    http://www.eclipse.org/org/documents/edl-v10.php.
+
+    SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+
+-->
+
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+    <id>common-license</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>${project.basedir}</directory>
+            <outputDirectory>md</outputDirectory>
+            <includes>
+                <include>LICENSE.md</include>
+                <include>NOTICE.md</include>
+            </includes>
+        </fileSet>
+        <fileSet>
+            <directory>${project.basedir}/features</directory>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>about.html</include>
+                <include>license.html</include>
+                <include>readme.html</include>
+            </includes>
+        </fileSet>
+    </fileSets>
+</assembly>

--- a/src/main/assembly/test-defaults.xml
+++ b/src/main/assembly/test-defaults.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -13,17 +13,21 @@
 
 -->
 
-<component xmlns="http://maven.apache.org/ASSEMBLY-COMPONENT/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-           xsi:schemaLocation="http://maven.apache.org/ASSEMBLY-COMPONENT/2.0.0 http://maven.apache.org/xsd/assembly-component-2.0.0.xsd">
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+    <id>test-defaults</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
     <fileSets>
         <fileSet>
-            <directory>${project.root.location}</directory>
+            <directory>${project.basedir}/etc</directory>
             <outputDirectory>.</outputDirectory>
             <includes>
-                <include>about.html</include>
-                <include>license.html</include>
-                <include>readme.html</include>
+                <include>*.properties</include>
             </includes>
         </fileSet>
     </fileSets>
-</component>
+</assembly>

--- a/testing/server-oracle/pom.xml
+++ b/testing/server-oracle/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -20,14 +20,14 @@
 
     <name>EclipseLink Test Server Parent</name>
     <groupId>org.eclipse.persistence</groupId>
-    <artifactId>org.eclipse.persistence.test.server.parent</artifactId>
+    <artifactId>org.eclipse.persistence.oracle.test.server.parent</artifactId>
     <packaging>pom</packaging>
 
     <parent>
-        <artifactId>org.eclipse.persistence.parent</artifactId>
         <groupId>org.eclipse.persistence</groupId>
+        <artifactId>org.eclipse.persistence.parent</artifactId>
         <version>3.0.3-SNAPSHOT</version>
-        <relativePath>pom.xml</relativePath>
+        <relativePath>../../pom.xml</relativePath>
     </parent>
 
     <build>

--- a/testing/server/pom.xml
+++ b/testing/server/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -20,14 +20,14 @@
 
     <name>EclipseLink Test Server Parent</name>
     <groupId>org.eclipse.persistence</groupId>
-    <artifactId>org.eclipse.persistence.oracle.test.server.parent</artifactId>
+    <artifactId>org.eclipse.persistence.test.server.parent</artifactId>
     <packaging>pom</packaging>
 
     <parent>
-        <groupId>org.eclipse.persistence</groupId>
         <artifactId>org.eclipse.persistence.parent</artifactId>
+        <groupId>org.eclipse.persistence</groupId>
         <version>3.0.3-SNAPSHOT</version>
-        <relativePath>pom.xml</relativePath>
+        <relativePath>../../pom.xml</relativePath>
     </parent>
 
     <build>

--- a/utils/eclipselink.dbws.builder.test.oracle.server/pom.xml
+++ b/utils/eclipselink.dbws.builder.test.oracle.server/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
 
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0 which is available at
@@ -27,7 +27,7 @@
         <groupId>org.eclipse.persistence</groupId>
         <artifactId>org.eclipse.persistence.oracle.test.server.parent</artifactId>
         <version>3.0.3-SNAPSHOT</version>
-        <relativePath>../../pom_oracle_test_server.xml</relativePath>
+        <relativePath>../../testing/server-oracle/pom.xml</relativePath>
     </parent>
 
     <properties>

